pytest/lib/proxy.py

```diff
@@ -326,6 +327,10 @@ def proxify_node(node, ps, handler, global_stopped, error, proxy):
     outer_port = inner_port + 100
 
     def start_proxy():
+        # local_stopped denotes the current of state of the proxy:
+        # 0: The proxy is running
+        # 1: The proxy is running but should be closed soon
```

Comment:
where is 1 used

Reply:
To kill a node, this value is set to `1` so the underlying proxy finishes

https://github.com/nearprotocol/nearcore/pull/3272/files#diff-6fc0b6294fe72a99a922a578c7bf0561L343